### DISCUSSION TITLE
Hint never goes away even after clicking "Got it".

### DISCUSSION THREAD

**Discussion by IamLizu** [community/community#5249] [Discussions]
So I asked a Q&A post on github/copilot-preview discussion. It is my first post there and the system was telling me about marking answers. 

![image](https://user-images.githubusercontent.com/26184316/130478345-b154febf-7098-4996-bf33-f7c62da4b1ac.png)

However, no matter how many times I clicked on <kbd>Got it!</kbd>, the hint stays there. It seems the button is not working.

**Comment by laymonage** [comment]
Related: https://github.com/github/feedback/discussions/5158

**Reply by IamLizu** [reply]
Indeed it is. However, I was able to mark as answer without any issue unlike the linked post.

**Comment by grahamperrin** [comment]
This is an **awful** bug! 

The persistence of the overlay makes it unreasonably difficult to use the _hidden_ **Update Discussion** button: 

![image](https://user-images.githubusercontent.com/192271/131240620-e388d497-ce9f-4e35-b67e-3f984a9d50ab.png)

I tried using <kbd>Tab</kbd> and <kbd>Space bar</kbd> to reach and use the **Got it!** button: 

![image](https://user-images.githubusercontent.com/192271/131240571-b9b91002-bb8b-4e12-a606-9c25f18e8254.png)

– not a workaround; the overlay persists. 

After around ten minutes, I managed to reach and use the hidden **Update Discussion** button by:

1. clicking the visible white space to the left of two invisible buttons
2. keying <kbd>Tab</kbd> _once_ to reach the _second_ invisible button
3. <kbd>Space bar</kbd>

**Reply by grahamperrin** [reply]
> 2. keying <kbd>Tab</kbd> _once_ to reach the _second_ invisible button

This is another bug; https://github.com/github/feedback/discussions/5344

**Reply by IamLizu** [reply]
Haha. I inspected an deleted the element from page when I wanted to update discussion. I am actually laughing.

**Reply by grahamperrin** [reply]
Another workaround: 

1. pretend to want to edit the answer beneath the discussion (use the hamburger from which the offending overlay popped up)
2. whilst the answer is in edit mode, click the visible **Update discussion** button
3. cancel the unwanted edition of the answer
4. the offending pop-up reappears.

**Reply by laymonage** [reply]
Or you can also use <kbd>Ctrl</kbd>+<kbd>Enter</kbd> to submit.

**Reply by IamLizu** [reply]
> Or you can also use Ctrl+Enter to submit.

Damn! Totally forgot that!
